var/spack/repos/builtin/packages/root/package.py

```diff
@@ -203,8 +209,13 @@ class Root(CMakePackage):
     # Qt4
     depends_on('qt@:4.999', when='+qt4')
 
-    # TMVA
+    # Python
+    depends_on('python@2.7:', when='+python', type=('build', 'run'))
     depends_on('py-numpy', when='+tmva')
+    # This numpy dependency was not intended and will hopefully
+    # be fixed in 6.20.04.
+    # See: https://sft.its.cern.ch/jira/browse/ROOT-10626
+    depends_on('py-numpy', when='@6.20.00:6.20.03 +python')
```

Comment:
Numpy should probably be type=build/run

Reply:
done. Also improved the other numpy depends_on.